w3d_translator/classes/object.py

```diff
@@ -0,0 +1,18 @@
+import xml.etree.ElementTree as ET
+
+
+class Object:
```

Comment:
I have a bit of an allergic reaction to naming objects `Object`. `object` is a keyword, and `Object` is very close to that. What about `Element` or `Node` or something?